src/web/mjs/connectors/MangaLivre.mjs

```diff
@@ -0,0 +1,113 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaLivre extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangalivre';
+        super.label = 'Manga Livre';
+        this.tags = [ 'manga', 'portuguese', 'webtoon' ];
+        this.url = 'https://mangalivre.com';
+    }
+
+    async _getMangas() {
+        let page = 1;
+        let morePages = true;
+
+        let request, data;
+        let mangas = [];
+
+        while ( morePages ) {
+            try {
+                request = new Request(new URL('/series/index/nome/todos?page=' + page, this.url), this.requestOptions);
+                data = await this.fetchDOM(request, 'div.seriesList ul.seriesList li > a.link-block');
+                page++;
+
+                for (const manga of data) {
```

Comment:
May utilize `map` instead of loop

Reply:
Didn't think I would like `.map` better. But the code looks much better when written as `.map`